nautobot/extras/models/groups.py

```diff
@@ -194,9 +194,20 @@ def _map_filter_fields(self):
                 logger.debug("Skipping %s: doesn't have a filterset field", missing_field)
                 continue
 
+            # Skip filter fields that have methods defined. They are not reversible.
+            if filterset_field.method is not None:
+                logger.debug("Skipping %s: has a filter method", missing_field)
+                continue
+
             # Get the missing model form field so we can use it to add to the filterform_fields.
             modelform_field = modelform_fields[missing_field]
 
+            # If `required=True` was set on the model field, pop "required" from the widget
+            # attributes. Filter fields should never be required!
+            if modelform_field.required:
+                modelform_field.required = False
+                modelform_field.widget.attrs.pop("required")
```

Comment:
Can we handle the Booleans ? Booleans have True/False, thus we should not save them into a filter unless they have a non default value.

Reply:
Isn't that what `NullBooleanField` is for?

Reply:
I haven't observed and issue with booleans?

Reply:
Okay I see what you mean now.